fabric-object-builder-api-v1/src/main/java/net/fabricmc/fabric/api/object/builder/v1/entity/FabricEntityTypeBuilder.java

```diff
@@ -148,6 +154,27 @@ protected FabricEntityTypeBuilder(SpawnGroup spawnGroup, EntityType.EntityFactor
 		return this;
 	}
 
+	/**
+	 * Sets the default attributes for a type of living entity.
+	 *
+	 * <p>This should not be called if your entity is not a {@link LivingEntity}.
```

Comment:
```suggestion
	 * <p>This should not be called if the entity type being built does not correspond to a {@link LivingEntity}.
```

Reply:
`does not correspond to` does sound a bit odd to me but it's dictionary meaning here makes sense.